src/express/api.js

```diff
@@ -18,8 +18,8 @@ class API {
     return response.data;
   }
 
-  getArticles({comments} = ``) {
-    return this._load(`/articles`, {params: {comments}});
+  getArticles({comments, offset, limit, filterOption} = {}) {
+    return this._load(`/articles`, {params: {comments, offset, limit, filterOption}});
   }
```

Comment:
```suggestion
  getArticles(params = {}) {
    return this._load(`/articles`, params);
  }
```
или оставить как есть при условии что нужен контроль за тем что попадет как параметры и этих параметров не будет становиться значительно больше (тогда нужно будет выносить в отдельный [адаптер](https://itnext.io/easy-patterns-adapter-9b5806cb346f))